flink-runtime-web/web-dashboard/bower.json

```diff
@@ -23,7 +23,8 @@
     "dagre-d3": "~0.4.10",
     "font-awesome": "~4.3.0",
     "moment-duration-format": "~1.3.0",
-    "qtip2": "~2.2.1"
+    "qtip2": "~2.2.1",
+    "highcharts-release": "~4.1.8"
```

Comment:
I'm already working on a change to remove only the charts. Apparently, `d3` would work for us because it is BSD-lincensed.